packages/moveable/src/icons/arrow_circle.svg

```diff
@@ -1,3 +1,3 @@
 <svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
-<path fill-rule="evenodd" clip-rule="evenodd" d="M25.5 8C25.5 7.72386 25.2761 7.5 25 7.5C24.7239 7.5 24.5 7.72386 24.5 8V12.849C23.1055 9.97825 20.12 8 16.6691 8C11.8912 8 8 11.7958 8 16.5C8 21.2042 11.8912 25 16.6691 25C17.8361 25 18.9506 24.7737 19.9684 24.3627C21.0616 23.9212 22.0426 23.267 22.8577 22.4525L22.1508 21.7452C21.43 22.4655 20.5618 23.0446 19.594 23.4354C18.6935 23.799 17.706 24 16.6691 24C12.4237 24 9 20.6323 9 16.5C9 12.3677 12.4237 9 16.6691 9C19.8183 9 22.5183 10.8549 23.7002 13.5H19C18.7239 13.5 18.5 13.7239 18.5 14C18.5 14.2761 18.7239 14.5 19 14.5H25C25.2761 14.5 25.5 14.2761 25.5 14V8Z" fill="currentColor"/>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M25.5 8C25.5 7.72386 25.2761 7.5 25 7.5C24.7239 7.5 24.5 7.72386 24.5 8V12.849C23.1055 9.97825 20.12 8 16.6691 8C11.8912 8 8 11.7958 8 16.5C8 21.2042 11.8912 25 16.6691 25C17.8361 25 18.9506 24.7737 19.9684 24.3627C21.0616 23.9212 22.0426 23.267 22.8577 22.4525L22.1508 21.7452C21.43 22.4655 20.5618 23.0446 19.594 23.4354C18.6935 23.799 17.706 24 16.6691 24C12.4237 24 9 20.6323 9 16.5C9 12.3677 12.4237 9 16.6691 9C19.8183 9 22.5183 10.8549 23.7002 13.5H19C18.7239 13.5 18.5 13.7239 18.5 14C18.5 14.2761 18.7239 14.5 19 14.5H25C25.2761 14.5 25.5 14.2761 25.5 14V8Z" fill="#ffffff"/>
```

Comment:
@samwhale What's wrong with `currentColor` here?

Reply:
Ah, forgot to change this back. Good catch 👍

Reply:
Added `currentColor` back in. If/when this svg is used as a react component or rendered within the same document it will be able to inherit the color from the parent element.